l10n_it_fiscal_document_type/tests/test_doc_type.py

```diff
@@ -49,3 +49,18 @@ def test_doc_type_update(self):
 
         invoice.type = 'out_refund'
         self.assertEqual(invoice.fiscal_document_type_id, self.TD04)
+
+    def test_doc_type_refund(self):
+        self.TD01.journal_ids = [self.journalrec.id]
+        self.TD01.refund_fiscal_document_type_id = self.TD04.id
```

Comment:
Avendo aggiornato il `fiscal.document.type.csv` , questa riga è inutile?

Reply:
dopo l'ultimo aggiornamento si :-), tolgo?

Reply:
mah sì, così in futuro non ci dobbiamo chiedere "come mai qui viene scritto `refund_fiscal_document_type_id`"?